Unity/Champloo/Assets/Scripts/Player/Movement States/Movement Specials/OnInfiniteDash.cs

```diff
@@ -36,8 +44,9 @@ public override MovementState DecideNextState(Vector3 velocity, Vector3 external
         if (isDisabled || timingState == TimingState.DONE)
         {
             hitDashEarly = false;
-            if (earlyAttackInput)
+            if (attackAllowed)
```

Comment:
My understanding was that we needed to put the player into the InAttack movement state as Player.cs doesn't do that.